### ISSUE TITLE
Run deployment on master

### ISSUE THREAD

**Issue by acrosson** [hjacobs/kube-aws-autoscaler#33]
Is there a way to run this on the master node, rather than a worker? I would prefer not to have an additional worker running if I don't need to. I would like to be able to scale from 0 nodes.

**Comment by hjacobs** [comment]
Yes, you should be able to run this on the master node --- the autoscaler will try to figure out all nodes and ASGs via K8s and AWS API. It does not care where it actually runs.
